lib/if.c

```diff
@@ -1106,7 +1106,7 @@ DEFPY_NOSH (interface,
 	 * interface is found, then a new one should be created on the default
 	 * VRF.
 	 */
-	VRF_GET_INSTANCE(vrf, vrfname, false);
+	VRF_LOOK_INSTANCE(vrf, vrfname, false);
```

Comment:
The commit comment does not make sense to me given what this commit does,  Can you clarify it?

Reply:
I was thinking VRF_GET_INSTANCE means: create vrf.
but actually, VRF_GET_ID does just a lookup.
so I propose to remove that commit.